docs/runbook/maintain-cardano-min-versions-file/README.md

```diff
@@ -0,0 +1,24 @@
+#  Cardano node minimum versions for Signers
+
+## Introduction
+
+The [cardano-min-versions.json](../../../cardano-min-versions.json) file, which specifies the minimum supported versions of the Cardano node for Mithril signers and facilitates the verification of compatibility for the Cardano networks.
```

Comment:
This should be updated with `networks.json`

Reply:
Done ✅